e2e-setup/deployment-templates/deployment-hibernate.yaml

```diff
@@ -0,0 +1,34 @@
+deployment:
+  configuration:
+    name: deploy-action-hibernate-e2e
+    description: ""
+    runtime_version: 12.1.0
```

Comment:
Will this version eventually become disallowed for new deployments?

Reply:
Eventually yes, but given that we are running these tests against dev/stage we could enable the LD flag to allow deprecated runtime version until we run into use case where deploy functionality varies based on runtime version